packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -485,7 +485,7 @@ TargetPlatform _currentHostPlatform(Platform platform) {
 
 HostPlatform _currentHostPlatformAsHost(Platform platform) {
   if (platform.isMacOS) {
-    return HostPlatform.darwin_x64;
+    return HostPlatform.darwin_x86;
```

Comment:
I think renaming this is going to inadvertently break some things

Reply:
CI agrees.